app/javascript/src/components/Common/Sidebar/constants.js

```diff
@@ -12,9 +13,17 @@ export const SIDENAV_LINKS = [
     to: "/notes",
     icon: Notes,
   },
+  {
+    label: "Contacts",
+    to: "/contacts",
+    icon: UserCircle,
+  },
   {
     label: "Settings",
     to: "/settings",
-    icon: Settings,
+    icon: Download,
   },
 ];
+
+export const AVATAR_IMAGE_URL =
+  " https://i.pinimg.com/originals/51/f6/fb/51f6fb256629fc755b8870c801092942.png";
```

Comment:
Why is this split into two lines? Was this auto-formatted? If not, make it a single line declaration.